src/compass/utils/h5_helpers.py

```diff
@@ -607,6 +614,7 @@ def metadata_to_h5group(parent_group, burst, cfg, save_noise_and_cal=True,
         add_dataset_and_attrs(burst_meta_group, meta_item)
 
     # Add parameters group in processing information
+    flag_set_applied = not burst.border[0].is_empty
```

Comment:
This flag gets turned on / off depending on what we have in `burst.border`. This is not a straightforward way of checking if SET was applied, but is the way I could think of to minimize the code modification. Any idea is appreciated.

Reply:
I think this is fine. Can you add a comment explaining why we are doing this (please, mention the issue with S1 historical data)

Reply:
Considering this as resolved because we are not going to take care of missing polygon issue for SET correction.